### DISCUSSION TITLE
打开订阅看是这样这是出什么问题啦

### DISCUSSION THREAD

**Discussion by pandamelive** [vvbbnn00/WARP-Clash-API#169] [General]
![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/59890056/785f07a1-ba2e-48c6-b511-6b7dbf3b4497)

**Comment by vvbbnn00** [comment]
能否提供一下控制台日志呢

**Comment by leetown3113** [comment]
删除templates挂载
